content/authors/authors.yml

```diff
@@ -21,3 +21,28 @@
       url: https://stackoverflow.com/story/zainfathoni
     - name: medium
       url: https://medium.com/@zain.fathoni
+
+
+- name: Darvin Sinaga
+  slug: darvin-sinaga
+  bio: |
+    Darvin adalah seorang Frontend Developer Tampan di prospace.io.
+    Saat ini ia sangat tertarik dan menekuni React dan  ekosistemnya.
+    Aktif di beberapa komunitas online maupun offline.
+
+    http://darvinsinaga.com
+  avatar: ./avatars/darvin.jpg
+  featured: true
+  social:
+    - name: github
+      url: https://github.com/nagacoder
+    - name: twitter
+      url: https://twitter.com/perajinkode
+    - name: facebook
+      url: https://www.facebook.com/darvin.sinaga1
+    - name: instagram
+      url: https://www.instagram.com/nagacoder
+    - name: linkedin
+      url: https://www.linkedin.com/in/darvinsinaga/
+    - name: medium
+      url: https://medium.com/@darvinsinaga
```

Comment:
```suggestion
      url: https://medium.com/@darvinsinaga

```
Tambahkan karakter `newline` di akhir baris

Reply:
`  url: https://medium.com/@darvinsinaga "\n" ` sperti ini?

Reply:
Bukan, seperti yang saya contohkan di atas.

Coba deh, buka [file ini](https://github.com/zainfathoni/pejuangkode/pull/7/files#diff-b437a62e3fb23c0c64723e45374de3f1R48), ada _icon_ merah di situ:
![image](https://user-images.githubusercontent.com/6315466/70856060-2b066a80-1f10-11ea-8f8d-c6d57e5481a8.png)

Untuk keterangan lebih lanjut, coba baca [jawaban StackExchange ini](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).